drivers/shared/executor/procstats/getstats.go

```diff
@@ -131,5 +131,6 @@ func (lps *linuxProcStats) StatProcesses() ProcUsages {
 	}
 
 	lps.cache = result
+	lps.at = time.Now()
```

Comment:
shouldn't this us `lps.clock`?

I have a slight preference for just specifying a now func on the struct and allow tests to override that instead of libclock. I think that would have made it easier to dodge this bug even, but it's not a blocker.

Reply:
Yeah, I like that too. Will fix.